mangum/handlers/abstract_handler.py

```diff
@@ -62,7 +61,7 @@ def api_gateway_endpoint_url(self) -> str:
     def from_trigger(
         trigger_event: Dict[str, Any],
         trigger_context: "LambdaContext",
-        **kwargs: Dict[str, Any],
+        api_gateway_base_path: str = "/",
```

Comment:
The `api_gateway_base_path` is always provided to `from_trigger`, so this does not require a default. I think for now just make this a positional argument.

Ideally it'd be good to refactor `from_trigger` to be more focused on only identifying the handler and accept the minimal event and context information required to do so (or be removed and handled at the adapter level instead), but that'd be too large of a change to do now.